src/MCPServer/share/mysql_dev.sh

```diff
@@ -1,37 +1,77 @@
 #!/bin/bash
 
-databaseName=$1
-dbpassword=$2  # should be "" or "-p<password>", like in localDevSetup/recreateDB.sh
```

Comment:
Old behavior was `$dbpassword` was `-p <password>` or `""`, so that it handled an empty password without requiring more input. This isn't preserved - `-p` always prompts, even when the password is empty, most notably when running the mysql_dev scripts.